src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculator {
```

Comment:
Don't put all behavior into a single interface if the methods are conceptually different from each other. Here, `Drawable` and `AreaCalculator` interfaces should not be implemented together if their methods are conceptually different.